templates/OperationOptionsDecodingSnippet.stencil

```diff
@@ -1,6 +1,6 @@
-{% if param.strategy == "dateFromConstant" %}
+{% if param.strategy == "dateFromParam" %}
 let {{ param.key }}String = DateFormatter().string(from: {{ param.key }})
-{% elif param.strategy == "byteArrayFromConstant" %}
+{% elif param.strategy == "byteArrayFromParam" %}
```

Comment:
Was this name changed because the param may not necessarily be a constant?

Reply:
Yes. I can be from one of the parametrs in the Option.